bot.py

```diff
@@ -68,6 +69,8 @@ def __init__(self):
         self.config = ConfigManager(self)
         self.config.populate_cache()
 
+        self.log_file_name = os.path.join(temp_dir, f"{self.token.split('.')[0]}.log")
+
```

Comment:
We no longer use this attribute. It has been removed and replaced with `self.log_file_path` (see line 92).